source/agora/consensus/validation/Transaction.d

```diff
@@ -129,8 +122,11 @@ public string isInvalidReason (
 
     Amount sum_unspent;
 
-    if (tx.type == TxType.Freeze)
+    if (tx.isFreeze)
     {
+        if (tx.outputs.count!(o => o.type == OutputType.Payment) > 1)
+            return "Transaction: Freeze cannot have multiple refund payment outputs";
+
```

Comment:
@MichaelKim20 Here is validation for more than single `Payment` output in a `Freeze` transaction.